backend/lib/cache/index.js

```diff
@@ -39,20 +45,27 @@ class Cache {
     }
   }
 
+  list (key) {
+    if (!this[key].isSynchronized) {
+      throw new CacheExpiredError(`Synchronization of "${key}" failed`)
```

Comment:
Resource out of sync. Please try again later.

Reply:
I have used this error message: The "projects" service is currently not available. Please try again later.